builtin/credential/cert/path_certs.go

```diff
@@ -147,6 +169,47 @@ func (b *backend) pathCertWrite(
 	policies := policyutil.ParsePolicies(d.Get("policies"))
 	allowedNames := d.Get("allowed_names").([]string)
 
+	// Parse the ttl (or lease duration)
+	systemDefaultTTL := b.System().DefaultLeaseTTL()
+	ttl := time.Duration(d.Get("ttl").(int)) * time.Second
+	if ttl == 0 {
+		ttl = time.Second * time.Duration(d.Get("lease").(int))
+	}
+	if ttl > systemDefaultTTL {
```

Comment:
Don't check against system TTL/max TTL. We'll automatically truncate the time for issued credentials and this can cause subtle issues. At the most, I'd make this a warning instead of an error.

Reply:
I've changed it to be a warning instead of error below.